changelogs/.plugin-cache.yaml

```diff
@@ -0,0 +1,34 @@
+plugins:
```

Comment:
Please remove this file

Reply:
Done

Reply:
doesn't look like removed to me, please verify again.

Reply:
Indeed, it was re added by mistake. Removed